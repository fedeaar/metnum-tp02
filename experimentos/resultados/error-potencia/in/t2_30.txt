6.100552000000000000e+06 -2.218307000000000000e+06 -7.129880000000000000e+05 -2.534150000000000000e+05 -9.552860000000000000e+05 -1.556318000000000000e+06 -2.318978000000000000e+06 -8.588010000000000000e+05 -7.289530000000000000e+05 2.550956000000000000e+06 2.637623000000000000e+06 -1.297892000000000000e+06 -2.252160000000000000e+05 8.056890000000000000e+05 -1.287760000000000000e+06 4.909300000000000000e+05 1.528079000000000000e+06 -1.835478000000000000e+06 -5.202190000000000000e+05 6.884000000000000000e+04
-2.218307000000000000e+06 6.071204000000000000e+06 6.627280000000000000e+05 -2.673419000000000000e+06 2.913200000000000000e+04 1.512083000000000000e+06 1.699323000000000000e+06 2.715496000000000000e+06 -3.069741000000000000e+06 -2.521629000000000000e+06 -2.125777000000000000e+06 1.130652000000000000e+06 -2.537114000000000000e+06 -1.094715000000000000e+06 -1.680818000000000000e+06 -5.469850000000000000e+05 7.430730000000000000e+05 -1.460579000000000000e+06 2.476838000000000000e+06 1.321904000000000000e+06
-7.129880000000000000e+05 6.627280000000000000e+05 5.119961000000000000e+06 -1.230420000000000000e+05 -7.242180000000000000e+05 -3.801200000000000000e+04 9.895400000000000000e+04 -1.398206000000000000e+06 1.759260000000000000e+05 8.270610000000000000e+05 -2.382031000000000000e+06 -1.089077000000000000e+06 -1.216917000000000000e+06 -2.061482000000000000e+06 1.697600000000000000e+06 -8.803900000000000000e+05 -1.028316000000000000e+06 1.367160000000000000e+05 -2.355080000000000000e+05 9.542940000000000000e+05
-2.534150000000000000e+05 -2.673419000000000000e+06 -1.230420000000000000e+05 6.787356000000000000e+06 -6.446700000000000000e+04 -1.379046000000000000e+06 2.190225000000000000e+06 -1.552473000000000000e+06 7.746820000000000000e+05 -9.973410000000000000e+05 4.000090000000000000e+05 -1.851036000000000000e+06 8.962480000000000000e+05 6.502790000000000000e+05 4.285894000000000000e+06 -1.480322000000000000e+06 -2.666137000000000000e+06 -1.680343000000000000e+06 -7.589140000000000000e+05 -4.762500000000000000e+04
-9.552860000000000000e+05 2.913200000000000000e+04 -7.242180000000000000e+05 -6.446700000000000000e+04 4.572935000000000000e+06 1.983991000000000000e+06 2.794462000000000000e+06 -2.822660000000000000e+05 -1.038624000000000000e+06 -8.947670000000000000e+05 -1.975042000000000000e+06 -7.426100000000000000e+04 4.455190000000000000e+05 1.643397000000000000e+06 1.247568000000000000e+06 -2.387460000000000000e+05 -3.907700000000000000e+05 1.872709000000000000e+06 3.612650000000000000e+05 5.595670000000000000e+05
-1.556318000000000000e+06 1.512083000000000000e+06 -3.801200000000000000e+04 -1.379046000000000000e+06 1.983991000000000000e+06 6.192739000000000000e+06 -3.842150000000000000e+05 1.415968000000000000e+06 -1.969225000000000000e+06 -1.093427000000000000e+06 -2.674244000000000000e+06 9.502920000000000000e+05 1.618855000000000000e+06 1.019049000000000000e+06 3.766430000000000000e+05 -8.050940000000000000e+05 -9.631920000000000000e+05 7.490380000000000000e+05 -1.111220000000000000e+06 1.447257000000000000e+06
-2.318978000000000000e+06 1.699323000000000000e+06 9.895400000000000000e+04 2.190225000000000000e+06 2.794462000000000000e+06 -3.842150000000000000e+05 8.456941000000000000e+06 -9.222980000000000000e+05 -5.542390000000000000e+05 -2.742463000000000000e+06 -1.068983000000000000e+06 -2.419328000000000000e+06 -2.039486000000000000e+06 7.484210000000000000e+05 1.779754000000000000e+06 -6.350880000000000000e+05 -1.105040000000000000e+05 8.720200000000000000e+04 2.971619000000000000e+06 5.665510000000000000e+05
-8.588010000000000000e+05 2.715496000000000000e+06 -1.398206000000000000e+06 -1.552473000000000000e+06 -2.822660000000000000e+05 1.415968000000000000e+06 -9.222980000000000000e+05 6.144919000000000000e+06 -1.386663000000000000e+06 -2.128569000000000000e+06 -2.211979000000000000e+06 3.658420000000000000e+05 5.184570000000000000e+05 1.057854000000000000e+06 -2.400225000000000000e+06 -9.768950000000000000e+05 1.126906000000000000e+06 8.585800000000000000e+04 1.435813000000000000e+06 1.892424000000000000e+06
-7.289530000000000000e+05 -3.069741000000000000e+06 1.759260000000000000e+05 7.746820000000000000e+05 -1.038624000000000000e+06 -1.969225000000000000e+06 -5.542390000000000000e+05 -1.386663000000000000e+06 6.162372000000000000e+06 6.830300000000000000e+04 1.417135000000000000e+06 -1.366824000000000000e+06 2.897830000000000000e+05 7.328320000000000000e+05 4.460910000000000000e+05 3.520960000000000000e+05 -1.615150000000000000e+05 1.683369000000000000e+06 -1.848681000000000000e+06 -2.446890000000000000e+05
2.550956000000000000e+06 -2.521629000000000000e+06 8.270610000000000000e+05 -9.973410000000000000e+05 -8.947670000000000000e+05 -1.093427000000000000e+06 -2.742463000000000000e+06 -2.128569000000000000e+06 6.830300000000000000e+04 8.104628000000000000e+06 6.592570000000000000e+05 -2.082536000000000000e+06 1.670291000000000000e+06 2.827190000000000000e+05 2.019400000000000000e+04 2.092383000000000000e+06 2.008886000000000000e+06 1.151863000000000000e+06 -1.958292000000000000e+06 -3.038533000000000000e+06
2.637623000000000000e+06 -2.125777000000000000e+06 -2.382031000000000000e+06 4.000090000000000000e+05 -1.975042000000000000e+06 -2.674244000000000000e+06 -1.068983000000000000e+06 -2.211979000000000000e+06 1.417135000000000000e+06 6.592570000000000000e+05 6.810768000000000000e+06 7.467020000000000000e+05 2.282780000000000000e+05 9.506060000000000000e+05 -4.950560000000000000e+05 1.021241000000000000e+06 2.513940000000000000e+05 -1.277403000000000000e+06 -9.919290000000000000e+05 -2.815380000000000000e+05
-1.297892000000000000e+06 1.130652000000000000e+06 -1.089077000000000000e+06 -1.851036000000000000e+06 -7.426100000000000000e+04 9.502920000000000000e+05 -2.419328000000000000e+06 3.658420000000000000e+05 -1.366824000000000000e+06 -2.082536000000000000e+06 7.467020000000000000e+05 7.639451000000000000e+06 -6.878990000000000000e+05 -1.599085000000000000e+06 -1.809062000000000000e+06 8.585190000000000000e+05 -1.933086000000000000e+06 -5.350340000000000000e+05 1.902843000000000000e+06 -1.989457000000000000e+06
-2.252160000000000000e+05 -2.537114000000000000e+06 -1.216917000000000000e+06 8.962480000000000000e+05 4.455190000000000000e+05 1.618855000000000000e+06 -2.039486000000000000e+06 5.184570000000000000e+05 2.897830000000000000e+05 1.670291000000000000e+06 2.282780000000000000e+05 -6.878990000000000000e+05 6.283216000000000000e+06 9.153460000000000000e+05 1.064746000000000000e+06 -8.020180000000000000e+05 -7.173650000000000000e+05 1.287647000000000000e+06 -2.782112000000000000e+06 8.178490000000000000e+05
8.056890000000000000e+05 -1.094715000000000000e+06 -2.061482000000000000e+06 6.502790000000000000e+05 1.643397000000000000e+06 1.019049000000000000e+06 7.484210000000000000e+05 1.057854000000000000e+06 7.328320000000000000e+05 2.827190000000000000e+05 9.506060000000000000e+05 -1.599085000000000000e+06 9.153460000000000000e+05 4.287675000000000000e+06 1.114919000000000000e+06 3.392380000000000000e+05 7.256480000000000000e+05 3.811830000000000000e+05 -1.070440000000000000e+05 1.332988000000000000e+06
-1.287760000000000000e+06 -1.680818000000000000e+06 1.697600000000000000e+06 4.285894000000000000e+06 1.247568000000000000e+06 3.766430000000000000e+05 1.779754000000000000e+06 -2.400225000000000000e+06 4.460910000000000000e+05 2.019400000000000000e+04 -4.950560000000000000e+05 -1.809062000000000000e+06 1.064746000000000000e+06 1.114919000000000000e+06 8.362257000000000000e+06 -2.146868000000000000e+06 -3.794444000000000000e+06 -1.924527000000000000e+06 -3.360837000000000000e+06 -4.767770000000000000e+05
4.909300000000000000e+05 -5.469850000000000000e+05 -8.803900000000000000e+05 -1.480322000000000000e+06 -2.387460000000000000e+05 -8.050940000000000000e+05 -6.350880000000000000e+05 -9.768950000000000000e+05 3.520960000000000000e+05 2.092383000000000000e+06 1.021241000000000000e+06 8.585190000000000000e+05 -8.020180000000000000e+05 3.392380000000000000e+05 -2.146868000000000000e+06 4.600573000000000000e+06 6.935600000000000000e+05 1.179780000000000000e+06 1.648093000000000000e+06 -2.217287000000000000e+06
1.528079000000000000e+06 7.430730000000000000e+05 -1.028316000000000000e+06 -2.666137000000000000e+06 -3.907700000000000000e+05 -9.631920000000000000e+05 -1.105040000000000000e+05 1.126906000000000000e+06 -1.615150000000000000e+05 2.008886000000000000e+06 2.513940000000000000e+05 -1.933086000000000000e+06 -7.173650000000000000e+05 7.256480000000000000e+05 -3.794444000000000000e+06 6.935600000000000000e+05 5.644911000000000000e+06 1.441508000000000000e+06 1.061862000000000000e+06 1.337120000000000000e+05
-1.835478000000000000e+06 -1.460579000000000000e+06 1.367160000000000000e+05 -1.680343000000000000e+06 1.872709000000000000e+06 7.490380000000000000e+05 8.720200000000000000e+04 8.585800000000000000e+04 1.683369000000000000e+06 1.151863000000000000e+06 -1.277403000000000000e+06 -5.350340000000000000e+05 1.287647000000000000e+06 3.811830000000000000e+05 -1.924527000000000000e+06 1.179780000000000000e+06 1.441508000000000000e+06 1.018641000000000000e+07 -8.206340000000000000e+05 8.100000000000000000e+02
-5.202190000000000000e+05 2.476838000000000000e+06 -2.355080000000000000e+05 -7.589140000000000000e+05 3.612650000000000000e+05 -1.111220000000000000e+06 2.971619000000000000e+06 1.435813000000000000e+06 -1.848681000000000000e+06 -1.958292000000000000e+06 -9.919290000000000000e+05 1.902843000000000000e+06 -2.782112000000000000e+06 -1.070440000000000000e+05 -3.360837000000000000e+06 1.648093000000000000e+06 1.061862000000000000e+06 -8.206340000000000000e+05 6.445047000000000000e+06 1.790910000000000000e+05
6.884000000000000000e+04 1.321904000000000000e+06 9.542940000000000000e+05 -4.762500000000000000e+04 5.595670000000000000e+05 1.447257000000000000e+06 5.665510000000000000e+05 1.892424000000000000e+06 -2.446890000000000000e+05 -3.038533000000000000e+06 -2.815380000000000000e+05 -1.989457000000000000e+06 8.178490000000000000e+05 1.332988000000000000e+06 -4.767770000000000000e+05 -2.217287000000000000e+06 1.337120000000000000e+05 8.100000000000000000e+02 1.790910000000000000e+05 5.557821000000000000e+06
