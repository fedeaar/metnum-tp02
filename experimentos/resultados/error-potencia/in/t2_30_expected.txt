2.214485037026327848e+07
1.992686185089819878e+07
1.496888101063022576e+07
1.295952889895244129e+07
1.155520561766207404e+07
1.035419159475901909e+07
8.717874495267713442e+06
5.561971153463172726e+06
4.819488435506629758e+06
4.178900369645904750e+06
3.535164449552135542e+06
3.063953133664341178e+06
2.496791605830984190e+06
1.899192586500575300e+06
1.366739879912011791e+06
9.908112761366962222e+05
4.791143958575286670e+05
4.117587233306365088e+05
7.815702095485698374e+04
2.229913121159291768e+04
